src/tests/stdlib/Mybuild

```diff
@@ -82,4 +82,9 @@ module strchr_test {
 	depends embox.framework.LibFramework
 }
 
+module ffs_test {
+	source "ffs_test.c"
 
+	depends embox.compat.libc.str
```

Comment:
Remove these dependencies please, we are trying to avoid test dependencies.

Reply:
Okay